django_q/queues.py

```diff
@@ -56,14 +55,13 @@ def __init__(self, *args, **kwargs):
 
         self.size = SharedCounter(0)
 
-    def put(self, *args, **kwargs):
-        super(Queue, self).put(*args, **kwargs)
+    def put(self, obj, block=True, timeout=None):
         self.size.increment(1)
+        return super().put(obj, block, timeout)
 
-    def get(self, *args, **kwargs):
-        x = super(Queue, self).get(*args, **kwargs)
+    def get(self, block=True, timeout=None):
         self.size.increment(-1)
-        return x
+        return super().get(block, timeout)
```

Comment:
Isn't using `super()` only giving problems with `Python 2.7`?
https://docs.python.org/2.7/library/functions.html#super
the `super()` call is introduced in `Python 3.0`

Reply:
Yep you are right, but I was just looking at some of the codacy issues and creating a pr was the easiest way to trigger it. On that point though, I do want to start creating a version that is Python 3 only. Even the Python gods will stop support for 2.7 in about 18 months.
I will probably start a python 3 only branch at some point soon.

Reply:
Love that idea!